.github/workflows/python-client-release-to-pypi.yml

```diff
@@ -0,0 +1,26 @@
+name: Release Python Client to pypi
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  run-python-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # We technically support 1.16 and 1.18; but this just needs any
+        # go version new enough to generate python proto files; 1.18 is
+        # more future-proof, so use it.
+        go: [ '1.18' ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/workflows/go-setup
+      - uses: ./.github/workflows/python-tests
```

Comment:
```suggestion
      - uses: ./.github/python-tests
```

Reply:
This is totally in the workflows/ directory, note how it matches the go-setup that's configured above it.

There were also syntax errors in CI logs because I didn't write the yaml for python-tests correctly the first time.